swagger.yml

```diff
@@ -961,10 +1594,6 @@ definitions:
 
   CreateOrUpdateClusterRequest:
     type: object
-    required:
-      - name
-      - providerSettings
-      - backupEnabled
```

Comment:
why this change?

Reply:
These fields are not actually required for updates.  But more importantly, for unpausing a cluster, we want to only send through the `Pause` field.  Sending other fields gave a 409 (Conflict) error "Cannot update cluster <name> while it is paused or being paused."

Reply:
👍